src/status/status.py

```diff
@@ -34,7 +34,7 @@
 SERVICES_DICT = {}
 
 ### SSL parameters
-USE_SSL = os.environ.get("F7T_USE_SSL", False)
+USE_SSL = os.environ.get("F7T_USE_SSL", "False") == "True"
```

Comment:
```suggestion
USE_SSL = os.environ.get("F7T_USE_SSL", False) == "True"
```
Why is it a string here, unlike the previous files? They have the same effect but maybe it's better to change all the files to be set in the same way?